tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def pytest_configure():
         },
         SITE_ID=1,
         SECRET_KEY='not_very_secret_in_tests',
-        WAGTAIL_SITE_NAME = 'test-site',
+        WAGTAIL_SITE_NAME='test-site',
```

Comment:
can you please separate formatting issues from this PR? they're not part of the problem you're solving.